### ISSUE TITLE
[fix] Accordion component fails a11y test when `multiselectable` is true

### ISSUE THREAD

**Issue by vim-usds** [trussworks/react-uswds#2204]
**ReactUSWDS Version & USWDS Version:**
ReactUSWDS: 3.1.0
USWDS: 2.13.3

**Describe the bug**
When setting the prop of `multiselectable` to true on the Accordion component, it fails the Axe Dev tools a11y check. When no setting this prop, the a11y check passes. 

**To Reproduce**

1. Install the Axe dev tools from [here](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US)
2. Render an Accordion component
3. Enable `multiselectable` to true
4. Open dev tools
5. Run devAxe on the page and error in screenshots section

**Expected behavior**
Expecting the component to pass a11y checks when `multiselectable` s true or false.

**Screenshots**

![Screen Shot 2022-10-13 at 8 58 14 PM](https://user-images.githubusercontent.com/86254807/195760132-9d2dddea-6db5-4d0a-a6e2-312a3d8928d8.png)

**Additional context**
No

**Device and Browser Information (please complete the following information if describing a UI bug):**

- Chrome 105
